fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -144,4 +144,12 @@
     <string name="translations_bottom_sheet_translating_in_progress">In progress</string>
     <string name="translations_bottom_sheet_translating_in_progress_content_description">Translating in Progress</string>
 
+    <string name="pref_automatic_translation_toolbar_title">Automatic translation</string>
```

Comment:
@ohall-m This is for a followup. Wondering if we are at the point of landing strings if we know the strings have been finalized?

Reply:
They should be finalized at this point except when otherwise noted! L10N has also been consulted too. `Copy: COMPLETE` and `L10N: COMPLETE` on the Figma and I also have that impression from other conversations.

Reply:
Yes, I agree that a big string PR is indeed very difficult to review and there is good reason to worry. However, if we continue to allow our static strings to grow bigger and land later. It will only fuel an even bigger string PR that would be even more difficult to review.

We can start small and open smaller PRs for strings in individual screens that have their strings finalized.

Reply:
Ahh, I see what you are saying, sounds good to me! I'll file some follow ups. I'm thinking`Core Translation Flow`, `Preferences`, `Errors`, and `Dialogs`. Can we land directly to `strings.xml` on patches like this or is that a bad practice?

Reply:
Perfect, thank you for the info!